c7n/resources/ebs.py

```diff
@@ -1002,6 +1007,8 @@ def get_volume_tags(self, volume, instance, attachment):
                 continue
             if t['Key'].startswith('aws:'):
                 continue
+            if t['Key'] in extant_tags and not overwrite:
+                continue
```

Comment:
this line doesn't seem to be covered by tests and is germane to the feature in the pr. afaics it should have been triggered per the assertion in the test on "self.assertEqual(tags["Testing"], "Original Value")"